### ISSUE TITLE
Ложные Problems

### ISSUE THREAD

**Issue by VlasProgulkin** [alliluja/RSL#15]
![image](https://user-images.githubusercontent.com/51082910/77427875-35c27c80-6de8-11ea-9281-e744568720f9.png)
Парсер нашел проблемы внутри комментария.

**Comment by alliluja** [comment]
Да, проблемы ищутся регэкспами без учета комментариев и строк. Возможно в будущем переделаю.
